docs/guides/access-groups.mdx

```diff
@@ -217,6 +219,44 @@ Finally, add users to the access group.  You can do this from the access group's
 You can only invite users who are already members of the account.  If you don't see a user on the Add users page, follow the steps in [Invite and manage users](/docs/guides/invite-and-manage-users#invite) to add them to the account first.  After they accept the invitation, you can add them to the access group.
 </Admonition>
 
+## Compare permissions
+
+The following table displays which permissions are granted to account owners and access groups.
+
+| Permissions | Account owner | Qiskit Runtime administrator (access group) | Instance collaborator (access group) |
+| --- | --- | --- | --- |
+| Full access to all IBM Cloud resources | | | | 
+| Assign access to others | | | | 
+| Create service instances  | | | | 
+| View all users | | | | 
+| Set user visibility | | | | 
+| Invite users to the account | | | | 
+| Billing responsibility | | | | 
+| View billing information | | | | 
+| Owner notifications | | | | 
+| Submit quantum workloads | | | | 
+| View others' quantum workloads | | | | 
+| Cancel others' quantum workloads | | | | 
+| Delete others' quantum workloads | | | | 
+| Create support cases | | | | 
+| Configure Account Single Sign-on (SS0) using IBM Cloud App ID | | | | 
+
+<span id="compare-actions"></span>
+## Compare pre-defined service role actions
+
+The following table displays which actions can be taken by the pre-defined service role actions: Manager, Editor, and Viewer.
```

Comment:
```suggestion
The following table displays examples of actions which can be taken by the pre-defined service role actions: Manager, Editor, and Viewer.
```

@abbycross maybe we can point to https://cloud.ibm.com/docs/account?topic=account-iam-service-roles-actions#actions-table152 for the full list of actions and roles mapping. And maybe we should follow the same format for the table that on that page.